integreat_cms/cms/forms/events/event_form.py

```diff
@@ -95,7 +96,21 @@ def clean(self):
         # make self.data mutable to allow values to be changed manually
         self.data = self.data.copy()
 
-        if cleaned_data.get("is_all_day"):
+        if (
+            cleaned_data["end_date"] - cleaned_data["start_date"]
+            > datetime.timedelta(6)
+            and not cleaned_data["is_recurring"]
+        ):
```

Comment:
```suggestion
        if cleaned_data["end_date"] - cleaned_data["start_date"] > timedelta(6):
```
I guess each occurrence of recurring events should also not take place longer than 7 days. Only the timedelta between the initial start date and the recurrence end date may be longer than 7 days.